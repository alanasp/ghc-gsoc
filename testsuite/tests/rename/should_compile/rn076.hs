{-# OPTIONS_GHC -fwarn-warnings-deprecations #-}

-- Test deprecation of variable and data constructor exports

module ShouldCompile where

import Rn076_A
import Rn076_B

-- funcRn076_C is deprecated at the defining module
-- and module Rn076_B but not Rn076_A, so we expect
-- only one warning

main = Rn076_B.funcRn076_C

main2 = funcRn076_C
