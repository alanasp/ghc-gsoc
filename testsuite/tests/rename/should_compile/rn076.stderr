rn076.hs:14:8: warning: [-Wdeprecations (in -Wdefault)]
    In the use of ‘funcRn076_C’
    (imported from Rn076_B, but defined in Rn076_C):
    Deprecated: "Rn076_C shall be imported directly"


